packages/orbit-design-tokens/src/dictionary/definitions/component/modal/modal-border.json

```diff
@@ -1,9 +1,26 @@
 {
   "component": {
+    "modal": {
+      "border-color": {
```

Comment:
hm, why we are adding this if we said, that the replacement for `borderColorModal` is `elevationFlatBackground` (I think the correct one is elevationFlatBorderColor, as it's for border and background is white). This was introduced for such kind of cases, where these borders will be used on multiple places and we do not need to have component tokens. So I think we can use for modal `elevationFlatBorderColor`

Reply:
Agreed. Will remove the token. But I think we can have a tailwind class that uses the token, so the class will be `border-modal`. But instead of using a token, we define on the TW config that `modal: tokens.elevationFlatBorderColor`.
I think it even looks better (currently the class was `border-modal-border-color`).

I'm not strongly against using the class `border-elevation-flat-border-color`, but I think that we have no harm in abstracting internally to `modal`. WDYT?

Reply:
Yep, it's no harm as you say, I just want it to be consistent, as for instance our Card after refactoring uses these flat elevations 🙂 
![Screenshot 2023-10-23 at 17 14 28](https://github.com/kiwicom/orbit/assets/14054752/1e161775-fff7-466f-93eb-c8572dc4d2dd)

Reply:
Hmm ok, I'll just go with that then 👍